src/onemod/backend/jobmon_backend.py

```diff
@@ -34,7 +34,6 @@
 from pathlib import Path
 from typing import Literal
 
-import yaml
```

Comment:
Highly recommend ruff pre-commit
https://github.com/astral-sh/ruff-pre-commit
it will automatically detect the unused import and fix it :)

Reply:
I had formatting on save but it wasn't working today for some reason. I had to update a vscode extension and now it is working again :)